spec/metatest_spec.rb

```diff
@@ -171,7 +171,7 @@ def compilation_board(expected_board = dummy_view_board)
         }
 
         it { expect(result[0][0]).to include :failed }
-        it { expect(result[0][0][2]).to include "The program was expected to fail by <strong>out of board</strong>, but it failed by another reason." }
```

Comment:
Well, apparently this line is returning out_of_board instead of out of board.
@flbulgarelli ?

Reply:
Yes, and I don't know why. This is not right

Reply:
Fixed. It was not just an extra `_` but a translation performed in the wrong place